platform/core/src/studies/studyUtils.js

```diff
@@ -0,0 +1,413 @@
+import DICOMWeb from '../DICOMWeb/';
```

Comment:
If this is the shared logic for the metadata loaders, would it make more sense as "internals" that live in the same location next to those files?

Reply:
Naming files "utilities" is a bit dangerous, in my opinion. I often think of utilities as general purpose static methods. These feel like data mappings, and a few methods for parsing data, building URLs, etc.

If it's simply shared logic you've tried to factor out, you can name it that, and put it close to the files that use it (if they're co-located). If it's generic data "mapping", then maybe we need to have a discussion (as a team) about creating a "mapping" layer where we discuss a more formal "Aggregate Root", "Viewmodel", and "Data Transfer Object" setup.

Reply:
ok changed it a bit..